_rest_routes.inc.php

```diff
@@ -604,6 +605,11 @@
         RestConfig::apiLog();
         return $return;
     },
+    "GET /fhir/metadata" => function () {
+        error_log("Route Meta");
+        RestConfig::authorization_check("patients", "demo");
+        return (new FhirMetaDataRestController())->getMetaData();
+    },
```

Comment:
```
    "GET /fhir/metadata" => function () {
        $return = (new FhirMetaDataRestController())->getMetaData();
        RestConfig::apiLog($return);
        return $return;
    },
```

Reply:
Above removes authorization_check (note this does not release it from auth check yet, but removes the acl check which shouldn't have) and logs it.

Reply:
To completely skip auth, add `/fhir/metadata` to this conditional:
https://github.com/openemr/openemr/blob/master/_rest_config.php#L178

To keep things clean, would rename that is_authentication() function to is_authentication_or_skip_auth() (or something to that effect) since the function is only called twice in the codebase.

Reply:
This is an API I created to test the FHIR Resource validation. It is an open API so anyone can use it.  

https://0zjitua3s4.execute-api.us-east-2.amazonaws.com/default/fhirResourceValidator

To use this API, once has to send a FHIR Bundle to this API.  Within the bundle we can send the Resource we are validating as an entry.

If the resource is valid, the Bundle will be returned, else the errors in the Resource will be sent.

I have been using this to test the FHIR Resources being generated by OpenEMR.  
Please feel free to use this , our community can also use this.